mlflow/R/mlflow/DESCRIPTION

```diff
@@ -55,12 +55,12 @@ Suggests:
     h2o,
     keras,
     lintr,
+    mleap,
     sparklyr,
     stringi (< 1.4.4),
     testthat (>= 2.0.0),
     xgboost
-Enhances:
-    mleap
+Additional_repositories: https://mlflow.github.io/drat
```

Comment:
Add comment say the drat repo is for mleap-R installation.

Reply:
`DESCRIPTION` file does not support comments. I added a comment in https://mlflow.github.io/drat instead.